server/export.go

```diff
@@ -72,13 +74,25 @@ func ExportCmd(ctx *Context, cdc *codec.Codec, appExporter AppExporter) *cobra.C
 			if err != nil {
 				return err
 			}
-			encoded = sdk.MustSortJSON(encoded)
-			fmt.Println(string(encoded))
+
+			var outputRaw json.RawMessage
+			codec.Cdc.UnmarshalJSON(encoded, &outputRaw)
+			outputRaw = sdk.MustSortJSON(outputRaw)
+
+			output, err := codec.MarshalJSONIndent(cdc, outputRaw)
+
+			outputFile := viper.GetString(flagOutputFile)
```

Comment:
Should we mark this flagOutputFile flag as required?

Reply:
It have default value